hudi-hadoop-mr/src/main/java/com/uber/hoodie/hadoop/HoodieInputFormat.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.uber.hoodie.hadoop;
+
+/**
+ * Temporary class to allow seamless migration of com.uber.hoodie to org.apache.hudi
+ */
+public class HoodieInputFormat extends org.apache.hudi.hadoop.HoodieInputFormat {
```

Comment:
@vinothchandar : this is a the remaining com.uber.hoodie input format

Reply:
this also is something we should note down to remove down the line.. for now, makes sense..

Reply:
Done